integration_tests/mockApis/orchestration.ts

```diff
@@ -53,6 +54,31 @@ export default {
       },
     })
   },
+  stubBookVisitValidationFailed: ({
+    visit,
+    validationErrors = ['APPLICATION_INVALID_NON_ASSOCIATION_VISITS'],
+  }: {
+    visit: Visit
```

Comment:
Don't actually need a `Visit` here; just `applicationReference: string` would do. This is because you don't return a `Visit` when the booking **fails** (unlike when a booking succeeds). 

Simplifying this means you don't need to set up and pass so much data when calling this stub in `checkYourBooking.cy.ts`.